src/schema.sql

```diff
@@ -36,30 +52,28 @@ CREATE TABLE bounds (
 CREATE INDEX bounds_gist ON bounds USING GIST(geom);
 CREATE INDEX bounds_idx ON bounds(name);
 
-DROP TABLE IF EXISTS users;
-CREATE TABLE users (
-    id          BIGSERIAL PRIMARY KEY,
-    access      TEXT,
-    username    TEXT UNIQUE NOT NULL,
-    password    TEXT NOT NULL,
-    email       TEXT UNIQUE NOT NULL,
-    meta        JSONB
-);
-
-DROP TABLE IF EXISTS users_tokens;
 CREATE TABLE users_tokens (
     name        TEXT,
-    uid         BIGINT,
+    uid         BIGINT REFERENCES users(id),
     token       TEXT PRIMARY KEY,
     expiry      TIMESTAMP,
     scope       TEXT
 );
 
-DROP TABLE IF EXISTS geo;
-DROP INDEX IF EXISTS geo_gist;
-DROP INDEX IF EXISTS geo_idx;
+CREATE TABLE deltas (
+    id          BIGSERIAL PRIMARY KEY,
+    created     TIMESTAMP,
+    features    JSONB,
+    affected    BIGINT[],
+    props       JSONB,
+    uid         BIGINT REFERENCES users(id),
```

Comment:
It's unclear to me that we really need a foreign key constraint here. We don't have them elsewhere, whats the thinking here?

Reply:
Not having them was an oversight on my part, we should definitely have references where possible to maintain database integrity.

Reply:
Removed in https://github.com/mapbox/Hecate/pull/206/commits/7493e0efafaeec0af7b70cbab56df3f1b16525d5 -- while these are changes we should make, i'm removing them from the current PR to reduce scope for what will already be a large change.